.github/workflows/release.yml

```diff
@@ -6,37 +6,107 @@ on:
+          auto_download_url="$(curl -fsSL https://api.github.com/repos/intuit/auto/releases/tags/$AUTO_VERSION | jq -r '.assets[] | select(.name == "auto-linux.gz") | .browser_download_url')"
           wget -O- "$auto_download_url" | gunzip > ~/auto
           chmod a+x ~/auto
 
+      - name: Check whether a release is due
+        id: auto-version
+        run: |
+          version="$(~/auto version)"
+          echo "::set-output name=version::$version"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  release:
+    runs-on: ubuntu-latest
+    needs: release-check
+    if: needs.release-check.outputs.auto-version != ''
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          path: dandischema
+
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
           python-version: 3.7
 
+      - name: Install dandischema
+        run: python -m pip install .
+        working-directory: dandischema
+
+      - name: Checkout dandi/schema
+        uses: actions/checkout@v2
+        with:
+          repository: dandi/schema
+          path: schema
+
+      - name: Test for unversioned changes
+        run: |
+          dirs="$(echo releases/*)"
+          echo "Following directories will be checked for differences: $dirs"
+          python ../dandischema/tools/pubschemata.py releases
+          git status
+          git add -A releases
+          if ! git diff --cached --exit-code $dirs
+          then echo "[ERROR] Existing schema files modified instead of creating a new version"
+               exit 1
+          fi
+        working-directory: schema
+
       - name: Install build & twine
         run: python -m pip install build twine
 
+      - name: Download auto
+        run: |
+          auto_download_url="$(curl -fsSL https://api.github.com/repos/intuit/auto/releases/tags/$AUTO_VERSION | jq -r '.assets[] | select(.name == "auto-linux.gz") | .browser_download_url')"
+          wget -O- "$auto_download_url" | gunzip > ~/auto
+          chmod a+x ~/auto
+
       - name: Create release
-        run: ~/auto shipit -vv
+        run: |
+          ~/auto shipit -vv
+          python -m build
+          twine upload dist/*
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           TWINE_USERNAME: __token__
           TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+        working-directory: dandischema
+
+      - name: Commit schema changes and create new tag
+        run: |
+          LIBRARY_VERSION="$(git -C ../dandischema describe --tags --exact-match)"
+          SCHEMA_VERSION="$(python -c 'from dandischema.consts import DANDI_SCHEMA_VERSION; print(DANDI_SCHEMA_VERSION)')"
+
+          git config --global user.email "team@dandiarchive.org"
+          git config --global user.name "DANDI Bot"
```

Comment:
good, harmonize then since there it is.  Lets keep this `DANDI Bot` one.  We should do that in dandi-cli as well
```
    "author": "auto <auto@nil>",
```

Reply:
Done.